test/blackbox-tests/test-cases/melange/emit-with-runtime-deps-dir-target.t

```diff
@@ -31,7 +31,9 @@ Rules created for the assets in the output directory
   [1]
 
   $ dune build @mel --display=short
+          bash some_dir
```

Comment:
In a separate PR, it would be good to switch this test to `sh` unless you need some special bash features.